src/examples/webvh-demo.ts

```
@@ -0,0 +1,99 @@
+/**
+ * Example: Creating a DID:WebVH identifier with the Originals SDK
+ * 
+ * This example demonstrates how to:
+ * 1. Create a new did:webvh DID with cryptographic signing
+ * 2. Save the DID log to the appropriate did.jsonl path
+ * 3. Load and verify the DID log
+ */
+
+import { WebVHManager } from '../did/WebVHManager';
+import * as path from 'path';
+import * as fs from 'fs';
```

ESLint

[error] 'fs' is defined but never used.

(@typescript-eslint/no-unused-vars)